tests/test_hierarchy_helpers.hpp

```diff
@@ -50,11 +51,30 @@ class Source final : public dealii::Function<dim>
 };
 
 template <int dim>
-class ConstantMaterialProperty final : public dealii::Function<dim>
+class Coefficient : public dealii::Function<dim>
```

Comment:
Isn't this more a `VectorizedFuntion` class or similar?

Reply:
Not really. The class is used for matrix-free where it is vectorized and the matrix-based which is not vectorized.